README.md

```diff
@@ -26,3 +39,47 @@ sudo make install
 sudo make uninstall
 ```
 
+## Standard installation
+
+Present SELinux policies are automatically applied when mongodb-server package is installed
+on a supported system.
+
+In order for mongod service to run, following assumptions are made:
+
+- daemon binary is installed in `/usr/bin/mongod`
+- database is located in `/var/lib/mongo`
+- log file must be located in `/var/log/mongodb/`
+- runtime data (PID) should be in `/var/run/mongodb/` or `/run/mongodb/`. On RHEL systems
+`/var/run` is a symbolic link to `/run`. This should not be changed
+- default unix socket file goes to `/tmp`, which must stay a default location provided by
+operating system. It can not be a symbolic link to another location
+- default user created and configured by installer is used to run service
+- systemd unit file `/usr/lib/systemd/system/mongod.service` created by installer is used
```

Comment:
Can an unconfined user run `/usr/bin/mongod`?

Reply:
Should. We only transition mongod to mongod_t domain when launched from systemd:

```
type_transition initrc_domain mongod_exec_t:process mongod_t;
```